src/bms/lex/token.rs

```diff
@@ -381,14 +466,54 @@ impl<'a> Token<'a> {
                     Self::MidiFile(Path::new(file_name))
                 }
                 "#POORBGA" => Self::PoorBga(PoorMode::from(c)?),
-                "#VIDEOFILE" | "#MOVIE" => {
+                "#VIDEOFILE" => {
                     let file_name = c.next_line_remaining();
                     if file_name.is_empty() {
                         return Err(c.make_err_expected_token("video filename"));
                     }
                     Self::VideoFile(Path::new(file_name))
                 }
                 // Part: Command with lane and arg
+                // Place ahead of WAV to avoid being parsed as WAV.
+                #[cfg(feature = "minor-command")]
+                wavcmd if wavcmd.starts_with("#WAVCMD") => {
+                    let param = c
+                        .next_token()
+                        .ok_or_else(|| c.make_err_expected_token("wavcmd param (00/01/02)"))?;
+                    let param = match param {
+                        "00" => WavCmdParam::Pitch,
+                        "01" => WavCmdParam::Volume,
+                        "02" => WavCmdParam::Time,
+                        _ => return Err(c.make_err_expected_token("wavcmd param 00/01/02")),
+                    };
+                    let wav_index = c
+                        .next_token()
+                        .ok_or_else(|| c.make_err_expected_token("wavcmd wav-index"))?;
+                    let wav_index = ObjId::try_load(wav_index, c)?;
+                    let value = c
+                        .next_token()
+                        .ok_or_else(|| c.make_err_expected_token("wavcmd value"))?;
+                    let value: u32 = value
+                        .parse()
+                        .map_err(|_| c.make_err_expected_token("wavcmd value u32"))?;
+                    // Validity check
+                    match param {
+                        WavCmdParam::Pitch if !(0..=127).contains(&value) => {
+                            return Err(c.make_err_expected_token("pitch 0-127"));
+                        }
+                        WavCmdParam::Volume if value > 100 => {
+                            return Err(c.make_err_expected_token("volume 0-100"));
+                        }
```

Comment:
Maybe the volume parameter can be greater than 100. It thought that was discouraged but allowed.